keyboards/dz60/keymaps/twschum_b_4_10_simple/keymap.c

```diff
@@ -0,0 +1,81 @@
+#include "dz60.h"
+
+/* DZ60 layout using following options (from layouts diagram on KBDfans):
+ *  - plate B (2.25u lshift)
+ *  - opt 4 (1.75, 1, 1 on rshift)
+ *  - opt 10 (2.75, 1.25, 2.25 on space, 5x1u bottom right keys)
+ * http://www.keyboard-layout-editor.com/#/gists/225f0f4dcf6671405f744fabe314627c
+ */
+
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```

Comment:
This isn't needed, actually. 

```suggestion
```

`MOD_MASK_CTRL` is actually globally defined and does the same thing. 

Also, this doesn't appear to be used at all, anyways.